features/poller/api/src/main/java/org/opennms/netmgt/poller/MonitoredServiceTask.java

```diff
@@ -125,8 +125,8 @@ public String getLocation(){
      */
     public int calculateTaskTimeout() {
         BigDecimal taskTimeOut = BigDecimal.ZERO;
-        BigDecimal timeout =  m_parameters.get("timeout") == null ? BigDecimal.valueOf(1) : (BigDecimal)m_parameters.get("timeout");
-        BigDecimal retry = m_parameters.get("retry") == null ? BigDecimal.valueOf(1) : (BigDecimal)m_parameters.get("retry");
+        BigDecimal timeout =  m_parameters.get("timeout") == null ? BigDecimal.valueOf(300000) : (BigDecimal)m_parameters.get("timeout");
+        BigDecimal retry = m_parameters.get("retry") == null ? BigDecimal.valueOf(3) : (BigDecimal)m_parameters.get("retry");
```

Comment:
Hi Seth,

As discussed over the call, I changed the default timeout to 5mins and retry to 3.